benchmarks/benchmarks.py

```diff
@@ -0,0 +1,84 @@
+# Write the benchmarking functions here.
+# See "Writing benchmarks" in the asv docs for more information.
+import numpy as np
+import scipy.stats as st
+
+
+class Hist:
+	def time_histogram(self):
+		try:
+			data = np.random.rand(10000, 1000)
+			import numba
+
+			@numba.njit(cache=True)
+			def _hist(data):
+				return np.histogram(data, bins=100)
+			return _hist(data)
+		except ImportError:
+			data = np.random.rand(10000, 1000)
+			return np.histogram(data, bins=100)
+
+
+class Variance:
```

Comment:
Testing independent logic is a good start and makes it easy to see how numba should be implemented. Is it possible to also benchmark the arviz functions themselves with and without numba?

Reply:
This is an issue. This was my initial plan but importing arviz breaks the build and the benchmarks do not execute. In my notebook, the reduction in the timings of the arviz methods was somewhat proportionate to the reduction in the time of the modified function; so I thought that if importing arviz does not work then I can benchmark the modified methods to ensure that the things are going in the right direction.

Reply:
Strategy makes sense. Let us know how we can help you figure out how to keep the build from breaking. Doesn't need to be done on this pr